src/traces/image/plot.js

```diff
@@ -181,13 +174,21 @@ module.exports = function plot(gd, plotinfo, cdimage, imageLayer) {
             }
         })
         .then(function() {
-            var href, canvas;
+            var href, canvas, localStyle;
+            localStyle = initialStyle;
             if(trace._hasZ) {
                 canvas = drawMagnifiedPixelsOnCanvas(function(i, j) {return z[j][i];});
                 href = canvas.toDataURL('image/png');
             } else if(trace._hasSource) {
                 if(fastImage) {
                     href = trace.source;
+                    // Flip the SVG image as needed (around the proper center location)
+                    var axisScale = [(xa.range[0] < xa.range[1]) ? 1 : -1, (ya.range[0] < ya.range[1]) ? -1 : 1];
+                    var trans = '';
+                    trans += 'translate(' + (left + imageWidth / 2) + 'px,' + (top + imageHeight / 2) + 'px)';
```

Comment:
Let's use `Lib.strTranslate` here as we did in #5254.
There is no `Lib.strScaleXY` to handle different scales at the moment; so you should leave `scale(...` as is.

Reply:
Done